pkg/virt-launcher/virtwrap/cmd-server/server.go

```diff
@@ -415,6 +415,33 @@ func (l *Launcher) GetDomain(_ context.Context, _ *cmdv1.EmptyRequest) (*cmdv1.D
 
 	return response, nil
 }
+func (l *Launcher) GetQemuVersion(_ context.Context, _ *cmdv1.EmptyRequest) (*cmdv1.QemuVersionResponse, error) {
+
+	response := &cmdv1.QemuVersionResponse{
+		Response: &cmdv1.Response{
+			Success: true,
+		},
+	}
+
+	version, err := l.domainManager.GetQemuVersion()
+
+	if err != nil {
+		response.Response.Success = false
+		response.Response.Message = getErrorMessage(err)
+		return response, nil
+	}
+
+	if jVersion, err := json.Marshal(version); err != nil {
```

Comment:
`version` is just a string (`QEMU %d.%d.%d`?), why Marshal it to json?

Reply:
This is because the calling to these functions are performed through Remote Procedure Calls. Therefore, responses need to be serialized, I think that's why. Moreover, even if you can do this with the marshal/Unmarshall paperwork, IMHO it will be a bit inconsistent with the rest of the code.

Reply:
Yeah, I think you are right. It works without the marshal/unmarshal calls. Removed.